glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -294,6 +298,24 @@ definitions:
           - hour
           - day
 
+      memory_unit:
+        title: Memory unit
+        description: |
+          The unit that the incoming memory size values are recorded in.
+
+          The units are the power-of-2 units, so "kilobyte" is pedantically a "kibibyte".
```

Comment:
Well actually a more accurate adjective would be "correctly". \*pushes bridge of glasses up nose*

Reply:
Indeed.  That's probably a better (and less condescending) word to use.

Reply:
And here I was saying it as a joke.